src/shared/components/Spinner/Spinner.styles.ts

```diff
@@ -4,7 +4,7 @@ import { StyledFlexAllCenter } from 'shared/styles/styledComponents';
 import theme from 'shared/styles/theme';
 
 export const StyledSpinner = styled(StyledFlexAllCenter)`
-  position: fixed;
+  position: absolute;
```

Comment:
are you sure it does not affect appearance of the Spinner everywhere in project?